lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/ContextLifeCycle.cs

```diff
@@ -71,10 +71,10 @@ public void Start()
             IContextStart contextStart = new ContextStartImpl(Id);
             
             // TODO: enable
-            //foreach (IObserver<IContextStart> startHandler in _contextStartHandlers)
-            //{
-            //    startHandler.OnNext(contextStart);
-            //}
+            //// foreach (IObserver<IContextStart> startHandler in _contextStartHandlers)
```

Comment:
Why //// is required here? I've tried reducing it to //, and it raises no StyleCop violation.